libtrip-notification/src/main/java/com/mapbox/navigation/trip/notification/MapboxTripNotification.kt

```diff
@@ -156,13 +175,28 @@ class MapboxTripNotification constructor(
         }
     }
 
+    /**
+     * Creates [PendingIntent] for open application when notification view is clicked
```

Comment:
I suggest changing to
_Creates [PendingIntent] to open an application when ..._
or
_Creates [PendingIntent] for opening an application when ..._